components/Applicant/Jobs/ApplyModal.tsx

```
@@ -30,64 +33,209 @@ interface ApplyModalProps {
+      
+      // Close modal after a delay to show success message
+      setTimeout(() => {
+        onClose();
+        setIsSuccess(false);
+      }, 2000);
+      
+    } catch (error: any) {
+      console.error("Error submitting application:", error);
+      
+      // Check if the error is about already applying
+      if (error.message && error.message.toLowerCase().includes("already applied")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)